orbit/cmd/desktop/desktop.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/fleetdm/fleet/v4/pkg/open"
+	"github.com/getlantern/systray"
+)
+
+//go:embed favicon.ico
+var icoBytes []byte
+
+func main() {
+	// Our TUF provided targets must support launching with "--help".
```

Comment:
Oh because that's part of the checks after downloading a binary?

Reply:
Correct. Am thinking on having `"--check"` instead on all our executables that just exits successfully. (Principle of least surprise.)